tests/test_api_build.py

```diff
@@ -608,3 +609,20 @@ def test_noarch_foo_value():
     metadata = json.loads(package_has_file(fn, 'info/index.json').decode())
     assert 'noarch' in metadata
     assert metadata['noarch'] == "foo"
+
+
+@pytest.mark.xfail
```

Comment:
Please add an explanation of why this is an xfail.  You can do this as a comment, or as the ```reason``` argument to xfail.